CHANGELOG.md

```diff
@@ -1,34 +1,24 @@
 # Changelog rubocop-bitcrowd
 
-Presented in reverse chronological order.
+Presented in reverse chronological order. Document adheres to [keepachangelog.com](https://keepachangelog.com/en/1.0.0/), though our sections are called "Deprecations", "Potentially breaking changes", "New features", and "Fixes".
 
-## main
+## Unreleased
 
-https://github.com/bitcrowd/rubocop-bitcrowd/compare/v2.2.0...HEAD
+https://github.com/bitcrowd/rubocop-bitcrowd/compare/v2.3.0..HEAD
```

Comment:
Is it on purpose that you switched to `..` from `...`? The two dots means one will only see the file changes, not the commits.

Reply:
Oh really? Didn't know that. Thought the behaviour is akin to `git log`. I'll merely aligned these lines, but will change them back to `...` now.